src/vivarium_census_prl_synth_pop/results_processing/names.py

```diff
@@ -0,0 +1,26 @@
+from typing import Dict
+
+import pandas as pd
+from vivarium import Artifact
+
+from vivarium_census_prl_synth_pop.results_processing.formatter import (
+    format_data_for_mapping,
+)
+
+
+def get_first_name_map(
```

Comment:
I'd call this function `get_data_for_first_middle_name_mapping` and have it return `name_data`. Then have two new functions called `get_first_name_map` and `get_middle_name_map`. Those two function would call `get_data_for_first_middle_name_mapping` and then return empty dicts like this one does currently.